tox.ini

```diff
@@ -61,11 +59,16 @@ commands = sphinx-build -j auto -d {toxinidir}/doc/build/doctrees -b html {toxin
 
 [testenv:coverage]
 depends = py3
```

Comment:
since we are running the unit tests here now do we need to depend on py3 still?

Reply:
Don't know about the py3 depends, can take a look, but probably don't need it.

I can understand only wanting to run the unit tests once and generate coverage, but it means I can't run them locally and not generate coverage. I don't know what the best thing to do here is.

Reply:
> I can understand only wanting to run the unit tests once and generate coverage, but it means I can't run them locally and not generate coverage. I don't know what the best thing to do here is.

Yeah, it's not ideal. If there's a way to check if the 'coverage' environment is being queued in tox, we should be able to avoid running the coverage related stuff in the test run. I'll look into that.

Nevertheless, I believe we still should avoid running the tests again since it both extends the CI run duration and the default tox run's duration.